cluster-autoscaler/core/static_autoscaler_test.go

```diff
@@ -168,7 +168,8 @@ func TestStaticAutoscalerRunOnce(t *testing.T) {
 	}
 	context := NewScaleTestAutoscalingContext(options, &fake.Clientset{}, provider)
 	listerRegistry := kube_util.NewListerRegistry(allNodeListerMock, readyNodeListerMock, scheduledPodMock,
-		unschedulablePodMock, podDisruptionBudgetListerMock, daemonSetListerMock)
+		unschedulablePodMock, podDisruptionBudgetListerMock, daemonSetListerMock,
```

Comment:
Either all parameters in single line or each in separate one.

Reply:
We do that all over CA code. And in case of a dozen parameters that are just variables (like here) I don't like either of those - both 300 char lines and 20 LOC to pass trivial parameters feel like a bad solution.

Reply:
Dunno, I've never seen any nice solution to this in golang.